docker/docker-compose.yml

```diff
@@ -364,7 +367,7 @@ services:
     profiles:
       - default
     extra_hosts:
-      - "host.docker.internal:host-gateway"
+      - "host.docker.internal:10.0.2.2"
```

Comment:
I'll revert this before an eventual merge. Leaving this comment here so I can't forget :)

Reply:
reverted in https://github.com/stacks-network/sbtc/pull/811/commits/f14503f8fc405b5995400433c7b477df82198cf3